WORKSPACE

```diff
@@ -38,6 +38,6 @@ load("@envoy_mobile//bazel:envoy_mobile_toolchains.bzl", "envoy_mobile_toolchain
 envoy_mobile_toolchains()
 
 # Fixing to API 29 since proguard seems to be failing for 30+
```

Comment:
This is outdated

Reply:
This is primarily to check if I can get an error on CI. We should use api version 21 instead of 24

(I don't have anything lower than 24 installed at the moment so I wanted a quick sanity check to make sure we _can_ build api 24)